Tests all clearhaus errors.

ishaimika opened the pull request:
Creates invalid authorizations to trigger diffrent kinds of [Clearhaus transaction status codes.](https://docs.gateway.clearhaus.com/#api-reference)

ishaimika commented:
Still no unit tests.

fredrikbryntesson commented on Test/3dproblem.ts:
```diff
@@ -0,0 +1,26 @@
+		}
+		const token = connection && await Authorization.create(connection, creatable)
+		return gracely.client.malformedContent.is(token) &&
+			token.content.property == "card.pan" &&
```
I dont understand the name 3dproblem, and it should be 3Dproblem.

fredrikbryntesson commented on Test/addauthentication.ts:
```diff
@@ -0,0 +1,26 @@
+			token.content.property == "card.pan" &&
+			token.content.description == "Invalid card number."
+	}
+})
```
addAuthentication is better but it should follow a pattern according to the two previous.

fredrikbryntesson commented on Test/blocked.ts:
```diff
@@ -0,0 +1,26 @@
+			token.content.property == "card.pan" &&
+			token.content.description == "Invalid card number."
+	}
+})
```
These files seems to ave the same content?

ishaimika commented:
> I dont understand this PR, why are we adding the same files with different names? How does this work? I dont see how the names are describing the test.

Changed PR comment to describe it better:
"Creates invalid authorizations to trigger diffrent kinds of [Clearhaus transaction status codes.](https://docs.gateway.clearhaus.com/#api-reference)"

Also added a description on all tests to correspond with Clearhaus transaction status codes.
